test/test-handler-spec.js

```diff
@@ -9,7 +9,7 @@ chai.should();
 import TestHandler from '../src/lib/test-handler';
 import options from './fixtures/options';
 
-const timeoutMs = 12000;
```

Comment:
I found this timeout too short for my shady old laptop. CI was also very close to hitting the 12 sec limit, so I increased to something that is still reasonable.